src/styles/index.scss

```diff
@@ -1,3 +1,75 @@
+@import './utils/variables';
+@import './utils/mixins';
+
 body {
   margin: 0;
 }
+
+.page {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  $duration: 60s;
+
+  width: $watch-size;
+  height: $watch-size;
```

Comment:
The variables `$watch-size`, `$minute-w`, and `$minute-h` are used here. Ensure these variables are defined in the imported files or within this file to avoid compilation errors.